includes/resources/pfbc/Element/Checkbox.php

```diff
@@ -40,6 +40,10 @@ public function render() {
 			$id = isset( $this->_attributes["id"] ) ? ' id="' . $this->_attributes["id"] . '-' . $count . '"' : '';
 
 			echo '<label class="' , $labelClass , ' ' , sanitize_title($value) , '">';
+			if($count < 1){
+                echo '<label for="'.$this->_attributes["name"].'" generated="true" class="error"></label>';
```

Comment:
@marin250189 What is this error class?

Reply:
If you inspect the HTML code you will se that the label with the error messages is only one.   Jquery validate allows to predefine where we want to place our error messages.  We just need to create a label with the right strcutre the for attribute value of the label  must match the name attribute of the input value. and the label class must be "error".